person.rb

```diff
@@ -24,7 +25,9 @@ def correct_name
   end
 
   def add_rentals(book, date)
-    @rentals << Rental.new(date, book, self)
+    @rentals = Rental.new(date, book, self)
+    @rentals << rental
+    book.add_rental(rental)
   end
```

Comment:
- [ ] Good job so far, but by just creating `Rental.new(date, book, self)` the book itself is sent as a parameter to create the new rental achieving the 'has-many' association` you don't need to add other lines of codes 🙃 🙃 
It can be like 👇🏾 

```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```